custodial-copy-reconciler/src/test/scala/uk/gov/nationalarchives/reconciler/MainSpec.scala

```diff
@@ -0,0 +1,113 @@
+package uk.gov.nationalarchives.reconciler
+
+import fs2.Chunk
+import org.scalatest.flatspec.AnyFlatSpec
+import org.scalatest.matchers.should.Matchers.{equal, should}
+import org.scalatest.{BeforeAndAfterAll, BeforeAndAfterEach}
+import software.amazon.awssdk.http.async.SdkAsyncHttpClient
+import software.amazon.awssdk.http.nio.netty.NettyNioAsyncHttpClient
+import uk.gov.nationalarchives.dp.client.Client.{BitStreamInfo, Fixity}
+import uk.gov.nationalarchives.dp.client.Entities.EntityRef.{ContentObjectRef, InformationObjectRef, NoEntityRef, StructuralObjectRef}
+import uk.gov.nationalarchives.dp.client.EntityClient.GenerationType.Original
+import uk.gov.nationalarchives.reconciler.Configuration
+import uk.gov.nationalarchives.reconciler.Main.Config
+import uk.gov.nationalarchives.reconciler.TestUtils.runTestReconciler
+import uk.gov.nationalarchives.utils.Detail
+import uk.gov.nationalarchives.utils.TestUtils.*
+
+import java.nio.file.Files
+import java.util.UUID
+
+class MainSpec extends AnyFlatSpec with BeforeAndAfterEach with BeforeAndAfterAll {
+
+  val databaseUtils = new DatabaseUtils("test-database")
+  import databaseUtils.*
+
+  override def beforeAll(): Unit = {
```

Comment:
If you make this `beforeEach` then we won't get clashes with tests polluting the database.

You can also get rid of the `BeforeAndAfterAll` trait if you do that.